src/layman/geoserver_proxy.py

```diff
@@ -186,6 +193,37 @@ def proxy(subpath):
                 ensure_wfs_t_attributes(wfs_t_attribs)
         except BaseException as err:
             app.logger.warning(f"WFS Proxy: error={err}, trace={traceback.format_exc()}")
+
+    query_params = CaseInsensitiveDict(request.args.to_dict())
+
+    # add buffer if incoming request is WMS GetMap in EPSG:3857 and one of SLD layers has native CRS EPSG:5514
+    # otherwise features are missing in the response in this specific case, we are not sure about the reason
+    if query_params.get('service') == 'WMS' and query_params.get('request') == 'GetMap' \
+            and (query_params.get('crs') or query_params.get('srs')) == crs_def.EPSG_3857:
+        layers = [layer.split(':') for layer in query_params.get('layers').split(',')]
+        use_buffer = False
+        for geoserver_workspace, layer in layers:
+            workspace = gs_wms.get_layman_workspace(geoserver_workspace)
```

Comment:
It can happen (by mistake or intentionally, e.g. when manually debugging) that user/we sends GetMap request with layer that is not in WMS workspace (`..._wms`) through this proxy. With current code it fails on assert in `get_layman_workspace` method, whereas I prefer to send unchanged request to the GeoServer, because it's valid request.

I suggest either to change the method get_layman_workspace that it returns input workspace if it does not end with `_wms`, or add some logic here.